Makefile

```diff
@@ -29,14 +29,28 @@ init:
 	@docker $(DOCKER_OPTS) commit $(CONTAINER_NAME) $(REPO)
 	@-docker $(DOCKER_OPTS) rm -f $(CONTAINER_NAME)
 
+node_modules: package.json
+	@npm install --quiet
+
+bower_components: node_modules bower.json
+	@npm run bower -- install $(BOWER_OPTS)
+
 run: PORT_MAP?=-p 8888:8888
 run: CMD?=jupyter notebook --no-browser --port 8888 --ip="*"
 run:
 	@docker $(DOCKER_OPTS) run --user root $(OPTIONS) \
 		$(PORT_MAP) \
 		-e USE_HTTP=1 \
 		-v `pwd`:/srv \
-		-v `pwd`:/root/.local/share/jupyter/nbextensions/declarativewidgets/urth_components/declarativewidgets-explorer \
+		-v `pwd`/elements:/root/.local/share/jupyter/nbextensions/declarativewidgets/urth_components/declarativewidgets-explorer \
 		--workdir '/srv/notebooks' \
 		--user root \
 		$(REPO) bash -c '$(CMD)'
+
+test: bower_components
+	@bower install ../widgets/elements/urth-core-behaviors/
+	@bower install ../widgets/elements/urth-viz-behaviors/
```

Comment:
Should probably make these paths to `declarativewidgets` since that is the official name of that repo.